src/openrct2/PlatformEnvironment.cpp

```diff
@@ -182,6 +186,7 @@ const char * PlatformEnvironment::FileNames[] =
     "hotkeys.dat",          // CONFIG_KEYBOARD
     "objects.idx",          // CACHE_OBJECTS
     "tracks.idx",           // CACHE_TRACKS
+    "RCTdeluxe_install" PATH_SEPARATOR "Data" PATH_SEPARATOR "mp.dat", // MP_DAT
```

Comment:
I think you're right - I forgot to re-add the check for non-Steam versions while changing a bunch of stuff. I'll fix it tomorrow.